src/web/routes/application/send-code/notify.js

```diff
@@ -0,0 +1,34 @@
+const { NotifyClient } = require('notifications-node-client')
+const uuid = require('uuidv4')
+const config = require('../../../../config')
+const notifyClient = new NotifyClient(config.environment.NOTIFY_API_KEY)
+const { logger } = require('../../../logger')
+require('dotenv').config()
```

Comment:
I don't think you need `dotenv` as you're not accessing a variable on `process.env`

Reply:
Removed.